### ISSUE TITLE
如何仅保存生成后的视频？

### ISSUE THREAD

**Issue by laidingq** [frankchieng/ComfyUI_Aniportrait#1]
大佬，我顺利的运行完工作流，看到保存的是视频带有原图片和面部网络，请问如果只保存生成后的视频？

**Comment by frankchieng** [comment]
> 大佬，我顺利的运行完工作流，看到保存的是视频带有原图片和面部网络，请问如果只保存生成后的视频？

nodes.py里头把video = torch.cat([ref_image_tensor, pose_tensor, video], dim=0)这行注释掉就可以了

**Comment by laidingq** [comment]
谢谢

**Comment by laidingq** [comment]
https://github.com/TMElyralab/MuseV  这个项目也是相关的，您是否有兴趣做成comfy插件
